src/exchange/billing/account-renew/billing-account-renew.template.html

```diff
@@ -0,0 +1,208 @@
+<div data-wizard
+    data-wizard-on-cancel="resetAction"
+    data-wizard-on-finish="submit"
+    data-wizard-title="'exchange_update_billing_action_title' | translate">
+
+    <div data-wizard-step data-wizard-step-valid="hasChanged()">
+
+        <p data-ng-bind-html="'exchange_update_billing_periode_intro' | translate"></p>
+
+        <div class="alert alert-warning" role="alert"
+            data-ng-if="!$ctrl.loading && $ctrl.bufferedAccounts && $ctrl.bufferedAccounts.list.messages.length > 0">
+            <div>
+                <strong data-translate="exchange_tab_ACCOUNTS_warning"></strong>
+                <span data-translate="exchange_tab_ACCOUNTS_partial"></span>
+            </div>
+        </div>
+
+        <form class="form-inline d-md-flex justify-content-md-end mb-3" name="searchAccountForm">
+            <label class="sr-only" for="accountSearch" data-translate="common_search"></label>
+            <div class="input-group">
+                <input
+                    type="text"
+                    class="form-control"
+                    id="accountSearch"
+                    maxlength="256"
+                    name="accountSearch"
+                    placeholder="{{::'exchange_tab_ACCOUNTS_table_email' | translate}}"
+                    data-ng-change="$ctrl.onSearchValueChange()"
+                    data-ng-disabled="$ctrl.loading"
+                    data-ng-model="$ctrl.search.value"
+                    data-ng-model-options="{ debounce: 800 }">
+                <div class="input-group-btn" data-ng-if="$ctrl.search.value">
+                    <button
+                        class="btn btn-default"
+                        type="button"
+                        aria-label="{{'common_cancel' | translate}}"
+                        data-ng-click="$ctrl.resetSearch()"
+                        data-ng-disabled="$ctrl.loading">
+                        <span class="fa fa-times" aria-hidden="true"></span>
+                    </button>
+                </div>
+                <span class="input-group-addon" data-ng-if="!$ctrl.search.value">
+                    <span class="fa fa-search" aria-hidden="true"></span>
+                </span>
+            </div>
+        </form>
+
+        <div data-ng-if="!$ctrl.exchange" class="text-center">
+                <oui-spinner></oui-spinner>
+        </div>
+
+        <div data-ng-if="$ctrl.exchange">
+
+            <div class="oui-datagrid-responsive-container">
```

Comment:
No `oui-datagrid` element possible ?

Reply:
Due to specific content in the `thead` it is not possible (but a revamp has been asked and will be taken care of)

Reply:
ok :)